src/components/forms/hooks/useControlledInput.ts

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+
+export default (
```

Comment:
please export hooks as named export
```js
export const useControlledInput = ...
```
this will enforce unified naming of hooks